### ISSUE TITLE
Indels are not allowed in LDpred

### ISSUE THREAD

**Issue by wavefancy** [bvilhjal/ldpred#21]
Hi @bvilhjal ,

It seems indels were excluded by the coord step, as the validated alleles were codes as ```valid_nts = set(['A','T','C','G'])```

Will this reduce the power of risk prediction, because there's a big proportion of indels in our genome, for example, 1/4 of variants in chr2 were excluded because of indels for the UK biobank imputed dataset. 

Best regards
Wallace

**Comment by bvilhjal** [comment]
Hi Wallace, 

Yes, indels are currently not included, which could lead to reduced power.  On the other hand, it is of course likely that some neighboring SNPs would capture the indels, and function as a proxy for them in practice.  Hence, I would expect the loss of prediction power to be limited.  

I'll keep this issue open, as this is clearly a good future improvement.

Best,
Bjarni

**Comment by anderseng** [comment]
I agree that this would be a great improvement.  Some of the most-associated loci in our GWAS are indels and some of them are not well-tagged by other SNPs in the vicinity.

**Comment by bvilhjal** [comment]
Hi @anderseng, 

Thanks for this suggestion.  Do you have an example of how those indels would be reported in the plink file?  Then I could perhaps address this.
